.github/CODEOWNERS

```diff
@@ -17,6 +17,7 @@
 /logging/               @googleapis/api-logging @googleapis/api-logging-partners @googleapis/yoshi-go-admins
 /profiler/              @googleapis/api-profiler @googleapis/yoshi-go-admins
 /vertexai/              @googleapis/go-vertexai @googleapis/yoshi-go-admins
+/internal/protoveneer/  @googleapis/yoshi-go-admins @jba @golang
```

Comment:
CODEOWNERS only works for individuals or with teams within the org of the repo.

Reply:
Do you have a suggestion for what I should write so that Eli and I can submit PRs?

Reply:
Oh, I see. I removed @golang.